ui/webui/test/helpers/installer.py

```diff
@@ -17,18 +17,40 @@
 
 
 class Installer():
+    _pages = ['installation-language',
+              'installation-destination',
+              'review-configuration',
+              'installation-progress']
+    _start_page = 'installation-language'
+
     def __init__(self, browser):
         self.browser = browser
 
-    def begin_installation(self):
+    def begin_installation(self, expecting_success=True):
+        current_page = self.get_current_page()
         self.browser.click("button:contains('Begin installation')")
+        if expecting_success:
+            self.wait_next_page(current_page)
 
-    def next(self):
+    def next(self, expecting_success=True):
+        current_page = self.get_current_page()
         self.browser.click("button:contains(Next)")
+        if expecting_success:
+            if current_page == self._start_page:
+                self.browser.expect_load()
```

Comment:
This will get obsolete from https://github.com/rhinstaller/anaconda/pull/3996 

I can remove it after, if this get's merged first.